info/CALCULATION.md

```diff
@@ -0,0 +1,191 @@
+# Description of calculations
+
+Let us introduce two basic concepts that are necessary for calculations: cycle length and period length.
+
+__Period length__ - days of release of blood from the uterus in a woman.
+__Cycle length__ - count of days from the first day of the current period to the first day of the next period. Period is part of the cycle. The cycle begins with it.
+
+The length of period usually 3-7 days, the length usually 21-35 days. These values may change each new cycle.
+
+## Calculation the expected start date of the next period
+
+Let's say we know the start date of the last period and average cycle length. Then, to calculate the start date of the next period you need:
+
+```ts
+function getNextPeriod(startDate: Date, cycleLength: number){
+    return startDate.addDays(cycleLength);
+}
+```
+
+But we only got the expected value, because the length of each new cycle is not always the same and this is normal. The length of the period may also change, so the length of the next period is calculated as the average value of previous ones.
+
+## Current day of cycle
+
+The current day of the cycle is important for next calculations. Depending on what day of the cycle it is, experience changes in the body (this is written below). This value is calculated as follows:
+
+```ts
+function getCurrentDay(startDate: Date, cycleLength: number){
+    const today = new Date();
+    return today.getDays() - startDate.getDays();
+}
+```
+
+## Ovulation day
+
+### What is ovulation
+
+__Ovulation__ is the release of a mature egg from an ovarian follicle into the abdomen. The egg waits 12-48 hours for fertilization. Further fertilization is impossible. So, only during ovulation can a woman become pregnant.
+
+Definition of the day of ovulation by calculations is only approximate. There are many reasons for this: ovulation may be early or late, there may be two, or there may not be, etc. You can find out exactly when ovulation occurred only with the help of special medical tests. But often ovulation coincides with calculations.
+
+### How to calculate the day of ovulation
+
+We know the value of the cycle length and the start date of the current cycle. For example, let the length of cycle is 30 days. In most cases, ovulation occurs 14 days before the start of your period (14 days before the start of a new cycle). Therefore, if the cycle length is 30 days, then ovulation will occur on ```30-14=16``` day of the cycle.
+
+Another example with dates:
+Let period (and therefore the cycle) begin on January 1, the average length of cycle is 25 days. Then ovulation will occur on the 11th day of the cycle, that is, January 11.
+
+So the function will look like this:

```

Comment:
will looks like this

Reply:
No, this is the future tense.
But I can rewrite this sentence in simple tense.